runtime/vm/ValueTypeHelpers.hpp

```diff
@@ -238,23 +238,32 @@ class VM_ValueTypeHelpers {
 	}
 
 	static VMINLINE bool
-	isClassRefQtype(J9ConstantPool *ramCP, U_16 cpIndex)
+	isNameOrSignatureQtype(J9UTF8 *utfWrapper)
 	{
-		J9ROMStringRef *romStringRef = (J9ROMStringRef *)&ramCP->romConstantPool[cpIndex];
-		J9UTF8 *classNameWrapper = J9ROMSTRINGREF_UTF8DATA(romStringRef);
-		U_16 classNameLength = J9UTF8_LENGTH(classNameWrapper);
-		U_8 *classNameData = J9UTF8_DATA(classNameWrapper);
 		bool rc = false;
+		if (utfWrapper)
+		{
```

Comment:
also if braces are on the same line `if (NULL != utfWrapper) {` if its a one condition if statement